styles/editorder.css

```diff
@@ -0,0 +1,87 @@
+/* InputField.CSS */
+
+.inputField {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 0.313rem;
+  margin-bottom: 0.8rem;
+}
+
+.inputField label {
+  line-height: 19px;
+}
+
+.inputField input,
+#amount {
+  background-color: hsla(0, 0%, 0%, 0);
+  color: var(--clr-white);
+  padding: 12px 16px;
+  margin: 4px 8px;
+  box-sizing: border-box;
+  width: 200px;
+  height: 43px;
+  border-radius: 7px;
+}
+
+.inputField input {
+  border: 2px solid var(--clr-white);
+}
+.inputField span {
+  visibility: hidden;
+  font-size: 0.75rem;
+}
+
+.valid span {
+  visibility: visible;
+  color: var(--clr-primary);
+}
+
+.invalid span {
+  visibility: visible;
+  color: var(--clr-error);
+}
+
+.valid input {
+  border: 2px solid var(--clr-primary);
+}
+
+.invalid input {
+  border: 2px solid var(--clr-error);
+}
+
+.inputField::placeholder {
+  color: var(--clr-placeholder);
+}
+
+.editorder {
+  padding-top: 4rem;
+  margin: auto;
+  width: fit-content;
+}
+
+#submit {
+  margin-bottom: 0.3rem;
+}
+#row_1,
+#row_3 {
+  line-height: 19px;
+  display: flex;
+  flex-direction: row;
+}
+#product_name {
+  width: 300px;
+}
+#editorderreset {
+  margin-left: 370px;
+  text-decoration: none;
+}
+#editordersubmit {
+  margin-left: 10px;
+}
+#editorder h1 {
+  margin-top: 40px;
+}
+.show {
+  display: flex;
+}
```

Comment:
Styling is close but doesn't quite match Figma, although not sure if thats a huge deal with the time we have remaining. Main thing is that the Product Field still looks intractable due to its colour, that could be worth changing.

Reply:
done